libs/net/dal/Configuration/SourceConfiguration.cs

```diff
@@ -27,8 +27,8 @@ public override void Configure(EntityTypeBuilder<Source> builder)
         builder.HasMany(m => m.Metrics).WithMany(m => m.Sources).UsingEntity<SourceMetric>();
         builder.HasOne(m => m.Product).WithMany(m => m.Sources).HasForeignKey(m => m.ProductId).OnDelete(DeleteBehavior.Restrict);
 
-        builder.HasIndex(m => m.Name, "IX_name").IsUnique();
-        builder.HasIndex(m => m.Code, "IX_code").IsUnique();
+        builder.HasIndex(m => m.Name, "IX_source_name").IsUnique();
```

Comment:
Renamed indexes because they were causing random mutations when we make new db migrations.